pkg/lambdaworks/lambdaworks.go

```diff
@@ -126,3 +130,23 @@ func (a Felt) Div(b Felt) Felt {
 	C.lw_div(&a_c[0], &b_c[0], &result[0])
 	return fromC(result)
 }
+
+// Writes the result variable with a << rhs.
+func (a Felt) Shl(rhs uint32) Felt {
+	for i := 0; i < N_LIMBS_IN_FELT; i++ {
+		a.limbs[i] <<= rhs
+	}
+	return a
+}
+
+func (a Felt) Bits() int {
+	if a.IsZero() {
+		return 0
+	}
+	val_u64, err := a.ToU64()
+	if err != nil {
+		fmt.Println(err)
+	}
+	return bits.Len64(val_u64)
+
+}
```

Comment:
We can use lambdawork's `UnsignedInteger::bits_le` method for this, this won't work for values bigger than u64